server/src/test/java/io/spine/server/aggregate/AggregateTest.java

```diff
@@ -835,33 +830,14 @@ void acknowledgeExceptionForDuplicateCommand() {
          * TaskAggregate#handle(AggAssignTask)}.
          */
         @Test
-        @DisplayName("in case of command dispatch")
+        @DisplayName("when dispatching a command")
         void fromCommandDispatch() {
-            BoundedContext boundedContext = newTaskBoundedContext();
-
-            TenantId tenantId = newTenantId();
-            Command command = command(createTask(), tenantId);
-            MemoizingObserver<Ack> observer = memoizingObserver();
-
-            boundedContext.getCommandBus()
-                          .post(command, observer);
-
-            assertNull(observer.getError());
-
-            List<Ack> responses = observer.responses();
-            assertSize(1, responses);
-
-            Ack response = responses.get(0);
-            io.spine.core.Status status = response.getStatus();
-            Error emptyError = Error.getDefaultInstance();
-            assertEquals(emptyError, status.getError());
-
-            Rejection emptyRejection = Rejection.getDefaultInstance();
-            assertEquals(emptyRejection, status.getRejection());
-
-            List<Event> events = readAllEvents(boundedContext, tenantId);
-            assertSize(1, events);
-            closeContext(boundedContext);
+            BlackBoxBoundedContext
+                    .with(new TaskAggregateRepository())
+                    .receivesCommand(createTask())
+                    .verifiesThat(acked(1).withoutErrorsOrRejections())
```

Comment:
I think that `acked` and `emitted` API methods need to be more clear to users and code-readers. Right now I am not even sure that I can 100% tell what they do. I can only try and guess.

Let's discuss in person.